src/scripts/dialogConfig.js

```diff
@@ -1,5 +1,24 @@
 import activateThebelab from './activateThebelab';
 
+function htmlToCode(str) {
+  return String(str)
+    .replace(/&amp;/g, '&')
+    .replace(/&lt;/g, '<')
+    .replace(/&gt;/g, '>')
+    .replace(/&quot;/g, '"')
+    .replace(/&#039;/g, '\'');
+}
+
+function codeToHtml(str) {
+  return String(str)
+    .replace('&', '&amp;')
+    .replace('<', '&lt;')
+    .replace('>', '&gt;')
+    .replace('"', '&quot;')
+    .replace('\'', '&#039;');
+}
```

Comment:
My hunch is that this isn't a fool proof way to do this. This has got to be something someone has solved in a robust and safe way. Maybe see if any JS packages handle this.

Reply:
I suppose this is one: https://www.npmjs.com/package/html-entities
Now safety wise, I think it would depend on whether or not the user can execute attacks while we are converting...

Reply:
As [underscore.js](https://github.com/jashkenas/underscore/blob/4334c126b3e71e4aeb8912d05d72139d24f7d0f1/underscore.js#L1514) also does it this way. We think this should be enough. We will also so add the back tick character.